admin/app/components/solidus_admin/ui/toggletip/component.erb

```diff
@@ -0,0 +1,26 @@
+<div class="relative inline-block" data-controller="<%= stimulus_id %>">
+  <%=
+    icon_tag(
+      "question-fill",
+      class: "w-[1rem] h-[1rem] #{icon_theme_classes} focus:outline-0 focus-visible:outline-1",
+      role: "button",
+      tabindex: 0,
```

Comment:
Yes, I tried, but sadly it broke the CSS. I added details in the commit message:

> We use a "button" role on the info icon instead of surrounding it with an actual button element because we found the latter to break the positioning of the speech bubble.

To be honest, I don't understand why, as it doesn't look as if new styles are added, so it's probably something related to the browser's default configuration. I'll spend a few minutes more trying to sort it out, as I'd love to use an actual button, but I'll timebox it to prevent losing too much time.

Reply:
Thanks for pushing me to try it again, @rainerdema! I was able to use a simple `button`, so now all that part to make the icon act like a button is gone, and the result is simpler. It turns out I needed to make the button a block element.